# --- T2-COPYRIGHT-BEGIN ---
# t2/package/*/chromium/hotfix-gperf32.patch
# Copyright (C) 2025 The T2 SDE Project
# SPDX-License-Identifier: GPL-2.0 or patched project license
# --- T2-COPYRIGHT-END ---

--- chromium-135.0.7049.84/third_party/blink/renderer/build/scripts/gperf.py.vanilla	2025-04-13 13:17:02.641204527 +0200
+++ chromium-135.0.7049.84/third_party/blink/renderer/build/scripts/gperf.py	2025-04-13 13:17:19.284151935 +0200
@@ -37,8 +37,8 @@
         # -Wimplicit-fallthrough needs an explicit fallthrough statement,
         # so replace gperf's /*FALLTHROUGH*/ comment with the statement.
         # https://savannah.gnu.org/bugs/index.php?53029
-        gperf_output = gperf_output.replace('/*FALLTHROUGH*/',
-                                            '  [[fallthrough]];')
+        #gperf_output = gperf_output.replace('/*FALLTHROUGH*/',
+        #                                    '  [[fallthrough]];')
         # -Wpointer-to-int-cast warns about casting pointers to smaller ints
         # Replace {(int)(long)&(foo), bar} with
         # {static_cast<int>(reinterpret_cast<uintptr_t>(&(foo)), bar}
